packages/react/src/Blankslate/Blankslate.module.css

```diff
@@ -0,0 +1,109 @@
+.Container {
+  container: blankslate / inline-size;
+}
+
+.Blankslate {
+  --blankslate-outer-padding-block: var(--base-size-32, 2rem);
+  --blankslate-outer-padding-inline: var(--base-size-32, 2rem);
+
+  display: grid;
+  justify-items: center;
+  /* stylelint-disable-next-line primer/spacing */
+  padding: var(--blankslate-outer-padding-block) var(--blankslate-outer-padding-inline);
+}
+
+.Blankslate[data-spacious='true'] {
+  --blankslate-outer-padding-block: var(--base-size-80, 5rem);
+  --blankslate-outer-padding-inline: var(--base-size-40, 2.5rem);
+}
+
+.Blankslate[data-border='true'] {
+  border: var(--borderWidth-thin) solid var(--borderColor-default);
+  border-radius: var(--borderRadius-medium);
+}
+
+.Blankslate[data-narrow='true'] {
+  max-width: 485px;
+  margin: 0 auto;
+}
+
+.Heading,
+.Description {
+  margin: 0;
+  /* stylelint-disable-next-line primer/spacing */
+  margin-bottom: var(--stack-gap-condensed, var(--base-size-8));
+}
+
+.Heading {
+  font-size: var(--text-title-size-medium, 1.25rem);
+  font-weight: var(--text-title-weight-medium, 600);
```

Comment:
We don't need to add fallbacks for the values here, the PostCSS parser inlines them for us.